### PR TITLE
Add ability to turn off user events prevention (close #1399)

### LINKED ISSUE #1399: Add ability to turn off user events prevention

### PR DESCRIPTION
**Pull request by churkin**
\cc @AlexanderMoskovkin @helen-dikareva

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

File: src/client/core/prevent-real-events.js [hunk 1]
```diff
@@ -70,7 +70,11 @@ function preventRealEventHandler (e, dispatched, preventDefault, cancelHandlers,
     }
 }
 
-export default function preventRealEvents () {
+export function preventRealEvents () {
     listeners.initElementListening(window, PREVENTED_EVENTS);
```
Comment (helen-dikareva) [inline src/client/core/prevent-real-events.js]: We should call `initElementListening` only once

Comment (helen-dikareva) [inline src/client/core/prevent-real-events.js]: Please check it

Comment (churkin) [inline src/client/core/prevent-real-events.js]: We can't do it because this method can be called twice for the window that will be recreated

Comment (helen-dikareva) [inline src/client/core/prevent-real-events.js]: ok

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Pull request merged**